scripts/lib/init.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Stolen from https://github.com/openshift/origin/blob/master/hack/lib/init.sh
+# to make life easier.
+function asb::absolute_path() {
+    local relative_path="$1"
+    local absolute_path
+
+    pushd "${relative_path}" >/dev/null
+    relative_path="$( pwd )"
+    if [[ -h "${relative_path}" ]]; then
+        absolute_path="$( readlink "${relative_path}" )"
+    else
+        absolute_path="${relative_path}"
+    fi
+    popd >/dev/null
+
+    echo "${absolute_path}"
+}
+readonly -f asb::absolute_path
+
+init_source="$( dirname "${BASH_SOURCE}" )/../.."
+ASB_ROOT="$( asb::absolute_path "${init_source}" )"
+ASB_PROJECT="ansible-service-broker"
+SCRIPT_DIR="${ASB_ROOT}/scripts"
+TEMPLATE_DIR="${ASB_ROOT}/templates"
+BROKER_TEMPLATE="${TEMPLATE_DIR}/deploy-ansible-service-broker.template.yaml"
+TEMPLATE_LOCAL_DEV="${TEMPLATE_DIR}/deploy-local-dev-changes.yaml"
+GENERATED_BROKER_CONFIG="${ASB_ROOT}/etc/generated_local_development.yaml"
+
+for library_file in $( find "${ASB_ROOT}/scripts/lib" -type f -name '*.sh' -not -path '*/scripts/lib/init.sh' ); do
+	source "${library_file}"
```

Comment:
this is indented via a tab. So we have 2 spaces, 4 spaces and now tabs for the shell scripts. -1

Reply:
My fault. I think this was the file I worked on that convinced me I needed to setup `ftplugin` for bash. I'll wait for the jury to return on 2v4 before I update.

Reply:
I'm still trying to figure out how you caught this.

Reply:
@djzager I caught it because in github diff it lines source up with the b in library. So that is 6 characters. I thought that was odd so I checked out the code and opened up the file. That's when I saw the tab :)